### DISCUSSION TITLE
Data output

### DISCUSSION THREAD

**Discussion by Pitcentr** [directus/directus#13362] [Archive]
How can I display data from another collection, there is a collection 1 and collection 2 are linked to each other by id, in collection 2 I see the id of the first, can you make it so that I would not only see the id of the first collection, and see all fields 1 collection in json format?

**Comment by br41nslug** [comment]
You can adjust the values shown for related fields either for the whole collection by setting the display template in your data model (under the "Collection Setup" header) or for a specific relationship by setting the display template in the interface when editing that field.

**Comment by Pitcentr** [comment]
In the API I get only the ID, what am I doing wrong ?

![изображение](https://user-images.githubusercontent.com/34168062/168833555-be158370-10b5-4456-a687-ec3aff3fb35d.png)

![изображение](https://user-images.githubusercontent.com/34168062/168833631-18e5ba51-5c72-4a2a-9b28-3d39148039dc.png)

![изображение](https://user-images.githubusercontent.com/34168062/168833735-a48c8dda-1fe0-48ac-be31-b3a3874985b4.png)

**Reply by br41nslug** [reply]
Sorry assumed you were talking about the App. When fetching through the API you need to specify which fields you want (https://docs.directus.io/reference/query/#fields) in you're example something like `?fields=*,country.*,city.*` should do the trick.

**Comment by Pitcentr** [comment]
Thanks for the answer, I'm trying but it still doesn't come out

![изображение](https://user-images.githubusercontent.com/34168062/168839397-46005d25-13ab-47ef-b380-a649a6f0a3b6.png)

**Reply by rijkvanzanten** [reply]
Also make sure that the user you're using (or the public role if you're not authenticating) has read access to the related collections 👍🏻

**Comment by Pitcentr** [comment]
thanks!!! works!!!

![изображение](https://user-images.githubusercontent.com/34168062/168841001-7a84bd57-2612-4497-9415-ba0ff7c5cb85.png)
